### PR TITLE
AOTS: Initial Article & Tournament Reorganisation

### PR DESCRIPTION
**Pull request by fajar13k**
Start of Asian-Oceanian Taiko Showdown article

---

### CODE REVIEW

File: wiki/Tournaments/AOTS/AOTS_1/en.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+- Asian-Oceanian Taiko Showdown
+- AOTS
+---
+# Asian-Oceanian Taiko Showdown
+
+![AOTS Logo](img/logo.png)
+
+> Uniting the mass diversity, NOW EXPANDED!
+
+The **Asian-Oceanian Taiko Showdown** (***AOTS***) is an Asian and Oceanian double-elimination one-on-one osu!taiko tournament hosted by ![][flag_ID] [fajar13k](https://osu.ppy.sh/users/7100002), ![][flag_US] [JDrago14](https://osu.ppy.sh/users/7690078), ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263). It is the third installment of the tournament and extension from the Asia region to Asia and Oceania.
```
Comment (TPGPL) [inline wiki/Tournaments/AOTS/AOTS_1/en.md]:
```suggestion
The **Asian-Oceanian Taiko Showdown** (***AOTS***) is an Asian and Oceanian double-elimination one-on-one osu!taiko tournament hosted by ![][flag_ID] [fajar13k](https://osu.ppy.sh/users/7100002), ![][flag_US] [JDrago14](https://osu.ppy.sh/users/7690078) and ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263). It is the third installment of the tournament and extension from the Asia region to Asia and Oceania.
```

File: wiki/Tournaments/AOTS/AOTS_1/en.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+| Semifinals | 2019-08-31/2019-09-01 |
+| Finals | 2019-09-08/2019-09-09 |
+| Grand Finals | 2019-09-15 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Unique profile badge, exclusive team banner |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | exclusive team banner |
```
Comment (TPGPL) [inline wiki/Tournaments/AOTS/AOTS_1/en.md]:
```suggestion
| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | Exclusive team banner |
```

File: wiki/Tournaments/AOTS/AOTS_1/en.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | exclusive team banner |
```
Comment (TPGPL) [inline wiki/Tournaments/AOTS/AOTS_1/en.md]:
```suggestion
| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | Exclusive team banner |
```

File: wiki/Tournaments/AOTS/AOTS_1/en.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+   - In Group Stage, you need to win 4 beatmaps to win a match. (Best-of-7)
+   - In the Round of 16 and the Quarterfinals, you need to win 5 beatmaps to win a match. (Best-of-9)
+   - In Semifinals, you need to win 6 beatmaps to win a match. (Best-of-11)
+   - In the Finals and Grand Finals, you need to win 7 beatmaps to win a match. (Best-of-13)
+
+### Match Instructions
+
+1. The referee will create the room and invite the respective players in.
+2. The participants will use `!roll` tournament to determine the order of picking beatmaps.
+3. The participants that scores higher will pick the first beatmap to play and second to bann.
```
Comment (WalterToro) [inline wiki/Tournaments/AOTS/AOTS_1/en.md]: Participant

File: wiki/Tournaments/AOTS/AOTS_1/en.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+4. The participants that scores lower will start banning the map first and picking map to play second.
```
Comment (WalterToro) [inline wiki/Tournaments/AOTS/AOTS_1/en.md]: Participant

File: wiki/Tournaments/AOTS/AOTS_1/en.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+2. Mappool consists of 9 maps at start of the stage and will increase by 2 maps by each stage.
+3. The NoMod bracket will be played with no mods activated.
+4. The FreeMod bracket will have FreeMod activated. Every individual player can pick Hidden, HardRock, both, or NoMod.
+5. The Tiebreaker will be played under FreeMod conditions.
+
+### Scheduling Instructions
+
+1. Each stage will be held on a **single weekend.**
+2. All stages will be held between Friday to Sunday, with UTC+7 as default timezone.
+3. The NoMod bracket will be played with no mods activated.
```
Comment (WalterToro) [inline wiki/Tournaments/AOTS/AOTS_1/en.md]: Uh? Duplicated and doesnt belong here

**Pull request merged**